### DISCUSSION TITLE
Quiz section in the sidebar

### DISCUSSION THREAD

**Discussion by H-a-r-s-h-a-S** [dailydotdev/daily#532] [Feature Requests]
I recently installed the daily.dev chrome exntesion and I find it very useful. Kudos to everyone who made this possible. I noticed that if there was a quiz/games section in the sidebar that one could just engage with to gain bits of knowledge in a entertaining manner, daily.dev would be even more awesome. Thank you.

**Comment by idoshamun** [comment]
Hi @H-a-r-s-h-a-S,

Thanks for the suggestion but this is right now out of our scope.
